neoload/commands/report.py

```diff
@@ -166,11 +166,20 @@ def get_resource_as_string(relative_path):
         # Try backported to PY<37 `importlib_resources`.
         import importlib_resources as pkg_resources
 
-    path = relative_path.split(os.path.sep)
+
+    path = split_path(relative_path)
     namespace = ".".join(path[:-1])
     file = path[-1]
     logging.debug({'path':path,'namespace':namespace,'file':file})
-    return pkg_resources.read_text(namespace, file)
+    contents = pkg_resources.read_text(namespace, file)
+    logging.debug("get_resource_as_string[contents]: {}".format(contents))
```

Comment:
Log the entire file in the logs is not a good idea. This won't help troubleshouting production issues, and the debug logs will be unreadable.
Remove this line